lib/api/compiler/class-beans-compiler.php

```diff
@@ -785,26 +785,28 @@ private function remove_modified_files( $hash, $filemtime_hash ) {
 	private function strip_whitespace( $content ) {
 		$replace = array(
 			'#/\*.*?\*/#s' => '', // Strip comments.
+			'#\s\s+#'      => ' ', // Strip excess whitespace.
```

Comment:
PUT BACK: @christophherr This is the original code that strips excess whitespace.